server/lib/genome/importers/file_importers/chembl.rb

```diff
@@ -0,0 +1,81 @@
+module Genome; module Importers; module FileImporters; module Chembl;
+  class Importer < Genome::Importers::Base
+    attr_reader :file_path
+    attr_accessor :chembl_data
+    def initialize(file_path)
+      @file_path = file_path
+      @source_db_name = 'ChEMBL'
+    end
+
+    def create_claims
+      query_chembl_db
+      create_interaction_claims
+    end
+
+    def create_new_source
+        @source ||= Source.create(
```

Comment:
indentation got a little wonky here -- I bet VSCode can autofix it (cmd+p -> '>Format' or something like that)